src/Mod/Path/PathScripts/PathJobGui.py

```diff
@@ -553,7 +554,9 @@ def setFields(self, obj):
         index = -1
         for i, solid in enumerate(self.candidates(obj)):
             self.form.stockExisting.addItem(solid.Label, solid)
-            if solid.Label == stockName:
+            label="%s-%s" % (self.StockLabelPrefix,solid.Label)
```

Comment:
`label="{}-{}".format(self.StockLabelPrefix, solid.Label)`

Reply:
I'm just playing with github.  I think there's a way to suggest a code change right in the web page and the author can accept it directly without having to do a roundtrip commit and push again.

Reply:
I can't use markdown to demo markdown, but it's like
```
'''suggestion
            label="{}-{}".format(self.StockLabelPrefix, solid.Label)
'''
```

Reply:
neat.  Thanks